src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Tests/Functions/CompleteNextDataSetVersionImportFunctionTests.cs

```diff
@@ -88,6 +88,14 @@ public async Task Success()
 
             Assert.Equal(originalDataSetVersionImport.Id, updatedDataSetVersionImport.Id);
 
+            var updatedDataSetVersion = publicDataDbContext
+                .DataSetVersions
+                .Single(import => import.Id == nextVersion.Id);
```

Comment:
Just a minor update needed here to use async and correct the variable name (it's a data set version, not an import). I think this should be

```csharp
var updatedDataSetVersion = await publicDataDbContext
                .DataSetVersions
                .SingleAsync(dsv => dsv.Id == nextVersion.Id);
```